### ISSUE TITLE
Exception raised in the DataPak helper

### ISSUE THREAD

**Issue by arifcatalyst** [bil-elmoussaoui/Hardcode-Tray#451]
### Specifications
- Hardcode-Tray version : `Hardcode-Tray: 4.1`
- Distribution : `Manjaro`
- Gtk version : `3.22`
- Inkscape version : `Inkscape 0.92.2 5c3e80d, 2017-08-06` if installed
- Cairo version : `` if installed
- Sni-qt version: `` if installed
- Application name :  `Papirus`
- Application version : ``

### Expected behavior and actual behavior
With `sudo -E hardcode-tray --conversion-tool RSVGConvert --size 22 --theme Papirus` 
I get the following error 

```
** (hardcode-tray:20008): WARNING **: Couldn't connect to accessibility bus: Failed to connect to socket /tmp/dbus-LjCeZlqjHo: Connection refused
Welcome to Hardcode-Tray!
Desktop Environment: Xfce
Scaling Factor: 1
Icon Size: 22
Icon Theme: Papirus
Conversion Tool: RSVGConvert
To Do: All
1 - Apply
2 - Revert
3 - Clear Backup Cache
Please choose:

```

**Comment by bil-elmoussaoui** [comment]
It's not an issue at all

**Comment by arifcatalyst** [comment]
How is this not an issue ? It doesn't work.

```
Please choose: 1
Applying now..


Traceback (most recent call last):
  File "/usr/bin/hardcode-tray", line 120, in <module>
    App.execute()
  File "/usr/lib/python3.6/site-packages/HardcodeTray/app.py", line 120, in execute
    delta = app.do_action(action)
  File "/usr/lib/python3.6/site-packages/HardcodeTray/modules/applications/application.py", line 151, in do_action
    self.install()
  File "/usr/lib/python3.6/site-packages/HardcodeTray/decorators.py", line 59, in wrapper
    func(app)
  File "/usr/lib/python3.6/site-packages/HardcodeTray/modules/applications/application.py", line 97, in install
    self.execute(Action.APPLY)
  File "/usr/lib/python3.6/site-packages/HardcodeTray/modules/applications/helpers/binary.py", line 51, in execute
    self.install_icon(icon, icon_path)
  File "/usr/lib/python3.6/site-packages/HardcodeTray/modules/applications/pak.py", line 66, in install_icon
    self.set_icon(icon, icon_path, png_bytes, True)
  File "/usr/lib/python3.6/site-packages/HardcodeTray/modules/applications/pak.py", line 48, in set_icon
    self.set_binary_file(path.join(str(icon_path), self.binary))
  File "/usr/lib/python3.6/site-packages/HardcodeTray/modules/applications/pak.py", line 44, in set_binary_file
    self.pak = DataPack(binary_file)
  File "/usr/lib/python3.6/site-packages/HardcodeTray/modules/applications/helpers/data_pack.py", line 32, in __init__
    self._read()
  File "/usr/lib/python3.6/site-packages/HardcodeTray/modules/applications/helpers/data_pack.py", line 66, in _read
    raise Exception
Exception

```

**Comment by bil-elmoussaoui** [comment]
Well, that wasn't what you have mentioned in the content of the issue
..

**Comment by arifcatalyst** [comment]
Since, I have mentioned now.

**Comment by bil-elmoussaoui** [comment]
The issue was closed before. Please report the right issue content next time;

**Comment by topisani** [comment]
I get this too, any ideas?

**Comment by sorrowless** [comment]
Have the same issue on latest ArchLinux installation.

**Comment by arifcatalyst** [comment]
Well, after the last update of` sni-qt`, I no longer have this issue. But Chromium-dev icon is still not fixed.And Chrome doesn't launch which is an existing bug.

**Comment by bil-elmoussaoui** [comment]
I'm pretty aware of this issue, but sadly I'm so busy these days to work on Hardcode-Tray. I will try to have some free time to do some fixes and create a new release 👍

**Comment by bil-elmoussaoui** [comment]
Can anyone of you guys send me the resources.pak file that you can find in /opt/google/chrome by email at bil[dot]elmoussaoui[at]gmail[dot]com
So i can work on this issue as soon as i can
Thanks

**Comment by varlesh** [comment]
rename **zip** to **pak**:
[resources.zip](https://github.com/bil-elmoussaoui/Hardcode-Tray/files/1463907/resources.zip)
Chrome 62.0.3202.89

**Comment by bil-elmoussaoui** [comment]
The issue is due to an outdated version of our modified version of data_pack helper (used to extract and create the pak file format).
Here's where the exception is raised 
https://github.com/bil-elmoussaoui/Hardcode-Tray/blob/559ea362dcbc8fbb84004c66072d2f7cc5807186/HardcodeTray/modules/applications/helpers/data_pack.py#L63-L65
And it's due to a non-supported version of Pak files, the version we support on stable is 4. The latest version of Chrome/Chromium uses a new version of pak files (version 5). Work already started, and there's still a small issue to fix; see here #411. So this issue should be fixed once I have fixed the other issue and created a new release.

For the time being, you can add google-chrome & chromium to the blacklist. See the README for more information about that.

I'm going to close this, as it's "already fixed" i just need to fix the other issue and create a hotfix release. 

Thanks!
